src/java/arjdbc/sqlite3/SQLite3RubyJdbcConnection.java

```diff
@@ -516,7 +513,7 @@ protected void setTimestampParameter(final ThreadContext context,
         final int index, IRubyObject value,
         final IRubyObject attribute, final int type) throws SQLException {
 
-        if (value instanceof RubyTime) value = ((RubyTime) value).strftime(TIMESTAMP_FORMAT);
+        if (value instanceof RubyTime) value = ((RubyTime) value).strftime(context, TIMESTAMP_FORMAT);
```

Comment:
did you check since what JRuby version the method exists?

Reply:
since 9.2.0.0 so we're good